src/SOS/SOS.UnitTests/Debuggees/GCWhere/GCWhere.cs

```diff
@@ -4,23 +4,26 @@
 // the object to pass though the GC generations until it hits
 // Gen 2 where it should stay.
 using System;
+using System.IO;
 using System.Diagnostics;
 using System.Runtime.InteropServices;
 
 class GCWhere
 {
     private string _string;
+    private static ulong _static = 52704621242434;
```

Comment:
Is a static unsigned int an interesting enough test?

Reply:
It is simple to verify the result.  The StringWriter variable and test is more interesting and actually tests that the bug is fixed.